Adding section for On-Demand Attachment Downloads (#587)
opengisch/QField-docs: documentation/get-started/tutorials/advanced-setup-qfc.en.md

!!! note
    This configuration must be set in the Settings page of each project in [QFieldCloud](https://app.qfield.cloud/).

## On-Demand Attachment Downloads

> For projects with many attachment files, you can enable on-demand downloading in QField.
> This is useful for saving storage space on field devices and reducing data transfer over limited network connections.
> 

Reviewer: ```suggestion
For projects with many attachment files, you can enable on-demand downloading in QField.
Enabling this function will not download attachments of other users or existing ones inside the QGIS projects.
Only the own attachments will be visible on the device.
This is useful for saving storage space on field devices and reducing data transfer over limited network connections.

```

> To enable this feature:

Reviewer: ```suggestion
### Activation of On-demand Attachment Download
```

1. Navigate to your project on the QFieldCloud web platform.
2. Open the project's **Settings** tab.
3. Enable the **On demand attachment files download** option.

!!! note

>     This feature can be activated during project creation or enabled at any time for existing projects.

Reviewer: ```suggestion
    This feature can be activated during project creation or later via the project settings directly in QFieldCloud at any given time.
```
